### PR TITLE
Remove add content sets plugin

### PR DESCRIPTION
**Pull request by ben-alkov**
Remove add_content_sets pre-build plugin

* CLOUDBLD-1135

Signed-off-by: Ben Alkov <ben.alkov@redhat.com>

# Maintainers will complete the following section

- [ ] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- [ ] Pull request has a link to an osbs-docs PR for user documentation updates
- [ ] New feature can be disabled from a configuration file

### CODE REVIEW

**Comment by ben-alkov** [comment]
osbs-box tests available from my [osbs-client PR](https://github.com/containerbuildsystem/osbs-client/pull/990#issuecomment-657789788)

File: docs/plugins.md [hunk 1]
```diff
@@ -179,10 +179,6 @@ containing the Dockerfile.
   - Status: Enabled
   - Markdown help page is converted to a man page and ADD'ed into the built
     image in order to show a correct page when `atomic help` is called.
-- **add_content_sets**
```
Comment (athos-ribeiro) [inline docs/plugins.md:182]: Have we replaced this docs when adding the new ICM plugin?

Comment (ben-alkov) [reply docs/plugins.md:182]: Fixed in PR #1510

**Comment by ben-alkov** [comment]
> LGTM, one detail: `constants.PLUGIN_ADD_CONTENT_SETS` could be removed too

:headdesk:

**Comment by athos-ribeiro** [comment]
Release notes added:
-  add content sets plugin removed, in favour of the new resove remote source plugin

**Pull request merged**